target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-rbs50.dts

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019-orbi.dtsi"
+
+/ {
+	model = "Netgear RBS50";
```

Comment:
Please use all-caps NETGEAR for model in all DTS files for consistency with existing devices.

Reply:
Fixed, thanks!